src/Files.App/Data/Contexts/DisplayPage/DisplayPageContext.cs

```diff
@@ -131,6 +131,20 @@ public DisplayPageContext()
 			layoutSettingsService.PropertyChanged += Settings_PropertyChanged;
 		}
 
+		public void UpdateAllTabsAndPanesLayout()
```

Comment:
```suggestion
		public void UpdateOpenTabsPreferences()
```

As a side note, should this method be under the page context? It seems it might be better as a helper.